src/hooks/useSwapCurrencyList.ts

```diff
@@ -381,12 +444,23 @@ const useSwapCurrencyList = (
     };
     doSearch();
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [searching, searchQuery, searchChainId, isCrosschainSearch]);
+  }, [
+    searching,
+    searchQuery,
+    searchChainId,
+    isCrosschainSearch,
+    inputCurrency?.type,
+  ]);
 
   const { colors } = useTheme();
 
   const currencyList = useMemo(() => {
     const list = [];
+    let bridgeAsset = isCrosschainSearch
```

Comment:
maybe we need this one to be in sync with `searchChainId` so we don't have mismatches

Reply:
no this is not happening, just checked

Reply:
Fix: https://github.com/rainbow-me/rainbow/pull/4274/commits/ec3d18865e698493656fb51e50ffe3260cf03ff7
https://recordit.co/olLhccighm